src/Components/CardboardList/CardboardListItem.tsx

```diff
@@ -102,6 +103,7 @@ export const CardboardListItem = <T extends unknown>(
                     {...buttonProps}
                     key={`cardboard-list-item-${listKey}-${index}`}
                     data-testid={`cardboard-list-item-${listKey}-${index}`}
+                    id={id}
```

Comment:
I think this should follow the same pattern as key and testid. with listKey and index.